File: openapi/code_samples/python/airtime/auto_detect_an_operator/code.py
import requests
import json

url = "https://topups.reloadly.com/operators/auto-detect/phone/03238482221/countries/PK"

payload={}
headers = {
  'Authorization': 'Bearer eyJraXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX',
  'Accept': 'application/com.reloadly.topups-v1+json'
}

response = requests.request("GET", url, headers=headers, data=payload)

print(response.text)

File: openapi/code_samples/nodejs/airtime/auto_detect_an_operator/code.js
var request = require('request');
var options = {
  'method': 'GET',
  'url': 'https://topups.reloadly.com/operators/auto-detect/phone/03238482221/countries/PK',
  'headers': {
    'Authorization': 'Bearer eyJraXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX',
    'Accept': 'application/com.reloadly.topups-v1+json'
  }
};
request(options, function (error, response) {
  if (error) throw new Error(error);
  console.log(response.body);
});